### PR TITLE
SumOfEvenValues

### PR DESCRIPTION
**Pull request by antonredko1993**

### CODE REVIEW

File: ElementaryTasks/Cicles.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (yuriygavriluk) [inline ElementaryTasks/Cicles.cs]: Remove empty lines

File: ElementaryTasks/Cicles.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ElementaryTasks
+{
+    class Cicles
+    {
```
Comment (GorbunowArtem) [inline ElementaryTasks/Cicles.cs:8]: Name should be 'Loops'

File: ElementaryTasks/Cicles.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        public void SumOfEvenValues()
+        {
+            int sum = 0;
+            int count = 0;
+            for (int i = 1; i < 99; i++)
+            {
+                if (i % 2 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Cicles.cs:15]: Always use brackets

File: ElementaryTasks/Cicles.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                    continue;
+
+                sum += i;
+                count += 1;
```
Comment (GorbunowArtem) [inline ElementaryTasks/Cicles.cs]: Just use count++

File: ElementaryTasks/Program.cs [hunk 5]
```diff
@@ -37,6 +37,10 @@ static void Main(string[] args)
                 75 - 89   B
                 90 - 100  A*/
             //logicalOperations.StudentEvaluation();
+
+            var cicles = new Cicles();
+
+            cicles.SumOfEvenValues();
```
Comment (GorbunowArtem) [inline ElementaryTasks/Program.cs]: No task provided. What should be done here?)

File: ElementaryTasks/Cicles.cs (hunk 4 above)
Comment (antonredko1993) [reply ElementaryTasks/Cicles.cs]: Done

File: ElementaryTasks/Program.cs (hunk 5 above)
Comment (antonredko1993) [reply ElementaryTasks/Program.cs]: fixed)

File: ElementaryTasks/Cicles.cs (hunk 2 above)
Comment (antonredko1993) [reply ElementaryTasks/Cicles.cs:8]: done

File: ElementaryTasks/Cicles.cs (hunk 3 above)
Comment (antonredko1993) [reply ElementaryTasks/Cicles.cs:15]: done

**Pull request merged**